hack/build-gitbooks.sh

```diff
@@ -18,9 +18,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-export KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
```

Comment:
This was never used anywhere, so there was no need to export it. Rather than clean it up to pass the linter (which it does not), I just used the same `cd` command we've been using across all our other projects.

Reply:
Sounds reasonable to me